api/src/main/kotlin/edu/wgu/osmt/RoutePaths.kt

```diff
@@ -66,3 +81,5 @@ object RoutePaths {
         const val COLLECTION_ID = "collectionId"
     }
 }
+
+
```

Comment:
remove one of these trailing return carriages

Reply:
removed